### DISCUSSION TITLE
[Idea] Error Logs with idempotency key by default

### DISCUSSION THREAD

**Discussion by justelis22** [PrestaShop/PrestaShop#31473] [Ideas]
### The solution

Logs could have an idempotency key by default that identifies logged operations during one action cycle. Object ID helps to how many operations were logged during one sequence. Timestamps are needed, as PrestaShop core logs check whether the same records are existing in the database. Using ms timestamps we are able to see the same messages and make sure each of them gets logged.

### Additional context

We have this done, so we could contribute

### Do you plan to work on this subject?

- [ ] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [ ] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [ ] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hello @justelis22 

Feel free to submit a Pull Request against the `develop` branch. It sounds like a good idea. There's a `date_add` column for the `log` table. I'm unsure whether we should have it modified (its type) or introduce a new field.
